Gemfile

```diff
@@ -7,8 +7,8 @@ if defined?(JRUBY_VERSION)
   gem 'json'
 elsif RUBY_VERSION =~ /^1/
   gem 'json', '~> 1.8.3'
-  gem 'tins', '~> 1.6.0'
-  gem 'term-ansicolor', '~> 1.3.0'
+  gem 'tins', '~> 1.15.0'
+  gem 'term-ansicolor', '~> 1.6.0'
```

Comment:
These aren't hard dependencies. Is there a particular reason we need these updated?